Sources/IndexStoreDB/IndexStoreDB.swift

```diff
@@ -200,6 +222,15 @@ public final class SymbolOccurrence {
   deinit {
     indexstoredb_release(value)
   }
+
+  private func getRelations() -> [SymbolRelation] {
+    var relations: [SymbolRelation] = []
+    indexstoredb_symbol_occurrence_relations(value){ relation in
```

Comment:
If we move the forEach onto the occurrence, we can implement this method in terms of that.